packages/choice-input/index.js

```diff
@@ -1 +1,2 @@
+export { ChoiceGroupMixin } from './src/ChoiceGroupMixin.js';
```

Comment:
For `checkbox` or `radio` we have different packages for the `choice` and the `group`. So why is it the same package for these?